addons/spreadsheet/static/src/global_filters/components/filter_value/filter_value.scss

```diff
@@ -8,3 +8,13 @@
         align-items: baseline;
     }
 }
+
+.date_filter_values {
+    select:has(option[value="empty"]:checked) {
```

Comment:
same comment than the 16.0 PR, this should be applied to lhe "last X days" filters as well :)